connaisseur/validators/notrayv1/notary.py

```diff
@@ -49,6 +50,28 @@ def __init__(self, name: str, host: str, pub_root_keys: list, **kwargs):
         self.pub_root_keys = pub_root_keys
         self.is_acr = kwargs.get("is_acr", False)
         self.is_cosign = kwargs.get("is_cosign", False)
+        self.auth = kwargs.get("auth", {})
```

Comment:
Previously, this included [logic to extract credentials from environment variables](https://github.com/sse-secure-systems/connaisseur/blob/47b60abc46b4d84546ad2e48883cc8501660e47c/connaisseur/notary_api.py#L170-L174). Why did that go away?

Reply:
Because I think putting things into a normal configfile and a "secret" configfile (one with sensitive data in it) and then merging them together, is a more modular approach then having a normal configfile and put in some specifi env variables in there, that depend on the type of configuration. We also don't really care what's in these config files, as long as the pass the schema validation.

Reply:
Ok. Such functional changes should ideally have their own commits, explaining in the commit message what's going on. :-)